subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -1391,14 +1409,16 @@ static void isr_rx_iso_data_invalid(const struct lll_sync_iso *const lll,
 	node_rx->hdr.handle = handle;
 
 	iso_meta = &node_rx->rx_iso_meta;
-	iso_meta->payload_number = lll->payload_count - bn - 1U;
+	iso_meta->payload_number = lll->payload_count - lll->bn - bn - 1U;
```

Comment:
`lll->bn - bn`

That looks odd. Can you explain why we do that? Preferably in a in-code comment

Reply:
True, its an error; the subtraction considered only one skipped ISO interval!... fixed now, also changed the decrementing `bn` while-loop to a for-loop now, which is hopefully more readable now.